library/hawkular_alerts_group_member.py

```diff
@@ -120,7 +122,25 @@ def group_member_exist(self, group_id, name):
             group_members = self.client.get_group_members(group_id)
         except Exception as e:
             self.module.fail_json(msg="Failed to get group members. Error: {error}".format(error=e))
-        return next((True for gm in group_members if gm.name == name), False)
+        return next((True for gm in group_members if gm.id == member_id), False)
```

Comment:
another q: if now you're looking by id, doesn't the client offer an efficient way to access only that member?